.github/workflows/tests.yml

```diff
@@ -138,6 +138,13 @@ jobs:
               console.log('No CI-build-dependents-from-source label found. Not passing --build-dependents-from-source to brew test-bot.')
             }
 
+            if (label_names.includes('CI-skip-online-checks')) {
```

Comment:
This makes me uncomfortable and is scope creep on what's discussed before. When would we pass this?

Reply:
This is case 3 from https://github.com/Homebrew/homebrew-test-bot/pull/699. In particular, I don't think we can merge #89042 without it.

Reply:
I am concerned this will be misused to ignore rather than fix these sort of checks. If we have annotations that produce warnings on PR: 👍🏻, otherwise I'm still 👎🏻 on this.

Reply:
(this feels like a slippery slope that ends up with `CI-skip-failing-tests`)

Reply:
I'm sympathetic to your concerns, but:

1. if we're contemplating just getting rid of these tests, it surprises me that we would be this worried about ignoring and not fixing them.
2. if a maintainer is set on ignoring failing CI, then there's not much we can do about that. This change just makes it so that, when that happens, we're at much less likely to be flying blind with respect to how much is broken by that decision.

That said, I'm fine with waiting until we can implement annotations that produce warnings.

Reply:
> 1. if we're contemplating just getting rid of these tests, it surprises me that we would be this worried about ignoring and not fixing them.

I think deleting is much preferable to ignoring. 

> 2\. if a maintainer is set on ignoring failing CI, then there's not much we can do about that. This change just makes it so that, when that happens, we're at much less likely to be flying blind with respect to how much is broken by that decision.

I think merging red CI or merging green CI with warnings is different to merging green CI which only passed because of a particular label.

> That said, I'm fine with waiting until we can implement annotations that produce warnings.

❤️ thanks!

Reply:
> That said, I'm fine with waiting until we can implement annotations that produce warnings.

Also, FYI, it seems to be just the matter of adding a `puts` with the relevant output:
https://github.com/Drieam/rspec-github/blob/c00ee4b3ba3051fc1e07b084c0f4ea544b455736/lib/rspec/github/formatter.rb#L18-L22

Reply:
Thanks for the hint -- I was wondering if that was all there is to it.